libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/VerifiableCredential.java

```diff
@@ -13,7 +13,7 @@
 import java.util.Date;
 
 @Getter
-@EqualsAndHashCode
+@EqualsAndHashCode(exclude = "signedJwt")
```

Comment:
Yeah, we use it now in the tests a bit when comparing VerifiableCredential instances. I added this because `SignedJwt` defers to Object for its equals method and so compares by identity, which isn't useful. I suspect because of signatures.
`JwtClaimsSet` on the other hand does a comparison of the claims `Map`, which will end up just comparing maps and strings I think, and seems to work.

Reply:
Cool, I don't think there's much value in comparing the `claimsSet` because it's purely derived from the `vcString`, but it doesn't matter very much if it's only used in tests.